sdcm/cluster_aws.py

```diff
@@ -417,20 +419,52 @@ def node_config_setup(self, node, seed_address, endpoint_snitch):
                               client_encrypt=self._param_enabled('client_encrypt'),
                               append_scylla_args=self.params.get('append_scylla_args'))
         else:
-            node.config_setup(enable_exp=self._param_enabled('experimental'),
-                              endpoint_snitch=endpoint_snitch,
-                              authenticator=self.params.get('authenticator'),
-                              server_encrypt=self._param_enabled('server_encrypt'),
-                              client_encrypt=self._param_enabled('client_encrypt'),
-                              append_scylla_args=self.params.get('append_scylla_args'))
+            if self.params.get('stop_service', default='true') == 'true':
```

Comment:
I also don't understand.

Reply:
It's not a new parameter, which was introduced by https://github.com/scylladb/scylla-cluster-tests/commit/65406f4353559bd16b1b988ce8d79421f3eb492e#diff-4542a95b1fea330a5c198c77a1b9dad2R335

When it's set to 'true', we will pass --stop-services in AMI user-data. Then the first start of scylla-server in AMI instance will fail as expected (won't start).

Because in our test, we have to reconfigure scylla in AMI instance. We want it really start after our second configuration.

Example: scylla-ami-setup will choose one seed, but we want to use multiple seeds.

More configuration we might reset after scylla-ami-setup finishes:
```
seed_address
experimental
endpoint_snitch
broadcast
authenticator
server_encrypt
client_encrypt
append_scylla_args
etc...
```

Reply:
@amoskong, I thought that in our latest changed of booting ami we always start with --stop-services.
Aren't we doing it? if not, why?